crates/sncast/src/starknet_commands/script/run.rs

```diff
@@ -274,138 +274,134 @@ impl<'a> ExtensionLogic for CastScriptExtension<'a> {
-        .casm_program()
-        .clone()
-        .assemble_ex(&entry_code, &footer);
-    let (hints_dict, string_to_hint) = hints_to_params(assembled_program.hints);
-
-    // hint processor
-    let mut context = build_context(
-        &SerializableBlockInfo::default().into(),
-        None,
-        &TrackedResource::CairoSteps,
-    );
-
-    let mut blockifier_state = CachedState::new(DictStateReader::default());
-
-    // TODO(#2954)
-    let param_types = builder.generic_id_and_size_from_concrete(&func.signature.param_types);
-
-    let segment_index = syscall_handler_offset(builtins.len(), has_segment_arena(&param_types));
-    let syscall_handler = SyscallHintProcessor::new(
-        &mut blockifier_state,
-        &mut context,
-        // This segment is created by SierraCasmRunner
-        Relocatable {
-            segment_index: segment_index
-                .try_into()
-                .expect("Failed to convert index to isize"),
-            offset: 0,
-        },
-        ExecutableCallEntryPoint::default(),
-        &string_to_hint,
-        ReadOnlySegments::default(),
-    );
-
-    let account = if config.account.is_empty() {
-        None
-    } else {
-        Some(tokio_runtime.block_on(get_account(
-            &config.account,
-            &config.accounts_file,
-            provider,
-            config.keystore.clone(),
-        ))?)
-    };
-    let state = StateManager::from(state_file_path)?;
-
-    let cast_extension = CastScriptExtension {
-        provider,
-        tokio_runtime,
-        config,
-        artifacts: &artifacts,
-        account: account.as_ref(),
-        state,
-    };
-
-    let mut cast_runtime = ExtendedRuntime {
-        extension: cast_extension,
-        extended_runtime: StarknetRuntime {
-            hint_handler: syscall_handler,
-            // If for some reason we need to calculate `user_args`, here
-            // is the function to do it: https://github.com/starkware-libs/cairo/blob/66f5c7223f7a6c27c5f800816dba05df9b60674e/crates/cairo-lang-runner/src/lib.rs#L464
-            // See TODO(#2966)
-            user_args: vec![vec![Arg::Value(Felt::from(i64::MAX))]],
-            panic_traceback: None,
-        },
-    };
-
-    match runner.run_function(
-        func,
-        &mut cast_runtime,
-        hints_dict,
-        assembled_program.bytecode.iter(),
-        builtins,
-    ) {
-        Ok(result) => match result.value {
-            RunResultValue::Success(data) => Ok(ScriptRunResponse {
-                status: "success".to_string(),
-                message: build_readable_text(&data),
-            }),
-            RunResultValue::Panic(panic_data) => Ok(ScriptRunResponse {
-                status: "script panicked".to_string(),
-                message: build_readable_text(&panic_data),
-            }),
-        },
-        Err(err) => Err(err.into()),
-    }
+    todo!("Use blockifier to run");
+    // warn_if_sncast_std_not_compatible(metadata)?;
```

Comment:
Will it be addressed?

Reply:
Nevermind, I see it's addressed in https://github.com/foundry-rs/starknet-foundry/pull/3246